.eslintrc.json

```diff
@@ -68,6 +68,8 @@
         "import/no-unresolved": 2,
         "import/no-unused-modules": 2,
         "import/no-useless-path-segments": 2,
+        "max-len": ["error", { "code": 130, "ignoreComments": false, "ignoreTrailingComments": false, 
+            "ignoreUrls": true, "ignorePattern": true, "ignoreRegExpLiterals": true}],
```

Comment:
Can you put these on one line? I know it isn't as readable, but it makes it possible to sort alphabetically when needed.

Reply:
Done. Also fixed the ordering.

Reply:
Thanks!